### ISSUE TITLE
关于损失函数

### ISSUE THREAD

**Issue by Yixan** [v-mipeng/LexiconNER#9]
您好，关于损失函数，在论文中提到：

> Therefore, in this work, we force l to be bounded by replacing the common unbounded cross entropy loss function with **the mean absolute error**, resulting in a bounded unbiased positive- unlabeled learning (buPU) algorithm. This slightly differs from the setting of uPU, which only requires ? to be slymmetric.

 我的理解是您论文中的buPU模型用MAE作为损失函数，但是在代码中定义的损失函数：

>     def loss_func(self, yTrue, yPred, type):
>         y = torch.eye(2)[yTrue].float()
>         if len(y.shape) == 1:
>             y = y[None, :]
>         # y = torch.from_numpy(yTrue).float().cuda()
>         if type == 'bnpu' or type == 'bpu':
>             loss = torch.mean((y * (1 - yPred)).sum(dim=1))
>         elif type == 'upu':
>             loss = torch.mean((-y * torch.log(yPred)).sum(dim=1))
>         # loss = 0.5 * torch.max(1-yPred*(2.0*yTrue-1),0)
>         return loss
是一个0-1损失而不是MAE。
求解惑，十分感谢！

**Comment by LUUUUUUZ** [comment]
请问一下这个问题或许有回答吗？为什么是简单的0-1损失呢？
